stat/init.lua

```diff
@@ -141,6 +141,18 @@ local function stat(args)
         stats['stat.net.sent.rps'] = box_stat_net.SENT.rps
         stats['stat.net.received.total'] = box_stat_net.RECEIVED.total
         stats['stat.net.received.rps'] = box_stat_net.RECEIVED.rps
+        if box_stat_net.CONNECTIONS ~= nil then
```

Comment:
Just in case: it is better to check whether the value is a table. See https://github.com/tarantool/tarantool/commit/68f43fa750bfcabb375199c2b28624761b0d8cba

Reply:
fixed

Reply:
Looks good, thanks!